crates/swc_ecma_codegen/src/lib.rs

```diff
@@ -718,17 +718,25 @@ where
         srcmap!(self, num, true);
 
         if self.cfg.minify {
-            value = minify_number(num.value);
-            self.wr.write_str_lit(DUMMY_SP, &value)?;
+            if num.value.is_infinite() && num.raw.is_some() {
+                self.wr.write_str_lit(DUMMY_SP, &num.raw.clone().unwrap())?;
```

Comment:
I'm not sure if I should be using `num.raw.as_ref().unwrap().as_str()` here or if there's any other way which can avoid the clone or if clone is fine here. Lemme know what should I use